conduct.md

```diff
@@ -100,32 +100,46 @@ TODO: Write an introductory paragraph.
 
 ### Conflicts
 
-TODO: Write an introductory paragraph.
+Students experiencing conflicts should come together to fix the problem. No argument
+should result in fighting, teams or groups should work together to solve any
+conflicts in the class. When dealing with conflicts if resolutions cannot be resolved
+then they should take conflicts to professor to resolve said issues. Lastly students
```

Comment:
Please change 'professor' to 'the Professor'. Additionally, please add a comma after 'Lastly'.